cells/cardano/nomadJob/default.nix

```diff
@@ -91,8 +91,9 @@ in
       namespace,
       datacenters ? ["eu-central-1" "eu-west-1" "us-east-2"],
       domain,
-      nodeClass,
+      nodeClass ? null,
```

Comment:
`nodeClass` is a `std`ized contract that we require from downstream clusters.

Making it optional would allow this contract to fail also in all other corcumstances.